apps/opik-frontend/src/components/pages-shared/experiments/useOptimizationScores.ts

```diff
@@ -63,6 +66,11 @@ export const useOptimizationScores = (
       }
     });
 
+    // If no experiment had a valid score, fallback to the most recent experiment
+    if (!retVal.bestExperiment && sortedRows.length > 0) {
```

Comment:
Thinking that the sortedRows.length check is redundant, as we already do that before, with experiments length, retVal

Also, maybe the check should be sortedRows.length, instead of experiments.length